docs/UserGuide.md

```diff
@@ -236,7 +236,7 @@ _Details coming soon ..._
 | **Add**    | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`|
 | **List**   | `list`|
 | **Edit**   | `editdetail [patient ID/detail field/updated patient details]`<br> e.g.,`editdetail 1/email/johndoe_updated@gmail.com`|
-| **Find**   | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`|
+| **search**   | `search KEYWORDS]`<br> e.g., `find James Jake`|
```

Comment:
Just stick with Find, makes more sense. Also the way you phrase it means can only find using one keyword